src/native-stack/views/NativeStackView.tsx

```diff
@@ -1,39 +1,41 @@
+import {
+  StackActions,
+  StackNavigationState,
+  useTheme,
+} from '@react-navigation/native';
 import * as React from 'react';
-import { View, StyleSheet, Platform } from 'react-native';
+import { Platform, StyleSheet, View } from 'react-native';
 import {
-  ScreenStack,
   Screen as ScreenComponent,
   ScreenProps,
+  ScreenStack,
 } from 'react-native-screens';
 import {
-  StackNavigationState,
-  StackActions,
-  useTheme,
-} from '@react-navigation/native';
-import HeaderConfig from './HeaderConfig';
-import {
-  NativeStackNavigationHelpers,
   NativeStackDescriptorMap,
+  NativeStackNavigationHelpers,
 } from '../types';
+import HeaderConfig from './HeaderConfig';
 
 const Screen = (ScreenComponent as unknown) as React.ComponentType<ScreenProps>;
+const isAndroid = Platform.OS === 'android';
 
 type Props = {
   state: StackNavigationState;
   navigation: NativeStackNavigationHelpers;
   descriptors: NativeStackDescriptorMap;
 };
 
-export default function NativeStackView({
+const NativeStackView: React.FC<Props> = ({
```

Comment:
Same as above.

Reply:
If it a case of preferring all of these to be reverted? Happy to do that and update the PR if so.